src/SFML/Window/OSX/SFWindowController.mm

```diff
@@ -99,7 +99,7 @@ - (id)initWithWindow:(NSWindow*)window
     {
         m_window        = nil;
         m_oglView       = nil;
-        m_requester     = 0;
+        m_requester     = nullptr;
```

Comment:
I don't know Objective-C++ well but do we want to use `nil` here?

Reply:
Yes, not sure why clang-tidy flagged that